pkg/scm/git/git.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strconv"
 	"strings"
 
+	log "github.com/golang/glog"
```

Comment:
glog is a wrapper in this file.  if you want to know the actual loglevel that's been set (as I do in the new logic), you need to use the real glog package to check.